sechub-sereco/src/main/java/com/mercedesbenz/sechub/sereco/importer/GitleaksSarifImportWorkaround.java

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.sereco.importer;
+
+import org.springframework.stereotype.Component;
+
+import de.jcup.sarif_2_1_0.model.ReportingDescriptor;
+import de.jcup.sarif_2_1_0.model.Run;
+import de.jcup.sarif_2_1_0.model.ToolComponent;
+
+@Component
+public class GitleaksSarifImportWorkaround implements SarifImportProductWorkaround {
+
+    @Override
+    public String resolveType(ReportingDescriptor rule, Run run) {
+        if (rule == null) {
+            return null;
+        }
+        if (isGitleaksRun(run)) {
+            return rule.getName();
+        }
+        return null;
+    }
+
+    private boolean isGitleaksRun(Run run) {
+        if (run == null) {
+            return false;
+        }
+        if (run.getTool() == null) {
+            return false;
+        }
+        ToolComponent driver = run.getTool().getDriver();
+        if (driver == null) {
```

Comment:
```suggestion
        /* only pseudo class - TheToolClass...*/
        TheToolClass tool = run.getTool()
        if (tool == null) {
            return false;
        }
        ToolComponent driver = tool.getDriver();
        if (driver == null) {
```

I would prefer a local variable here.